dataland-backend/src/main/kotlin/org/dataland/datalandbackend/api/CompanyApi.kt

```diff
@@ -81,14 +83,64 @@ interface CompanyApi {
     @PreAuthorize("hasRole('ROLE_USER')")
     fun getCompanies(
         @RequestParam searchString: String? = null,
-        @RequestParam dataTypes: Set<DataType>? = null,
+        @RequestParam(required = true) dataTypes: Set<DataType>,
```

Comment:
Change wird reverted nehme ich an, im Rahmen der Anpassung in der anderen conversation.

Reply:
Done